.github/workflows/presubmit.yml

```diff
@@ -196,6 +196,9 @@ jobs:
             cp '${{ env.builddir }}'/src/OpenCL.dll '${{ env.builddir }}'
           fi
           echo "testbindir=${{ env.builddir }}" >> $GITHUB_ENV
+      - name: Config tests
+        if: ${{ matrix.compiler-available && matrix.android-abi == '' }}
```

Comment:
We don't need the compiler-available or not to be on android for that test. Only the unit testing to be enabled.